WORKING_GROUPS.md

```diff
@@ -412,7 +413,7 @@ Responsibilities include:
   backporting changes to these branches.
 * Define the policy for what gets backported to release streams.
 
-### [Ecosystem Security](https://github.com/nodejs/security-wg)
```

Comment:
I think this was specifically changed to be Ecosystem Security

Reply:
@mhdawson addressed this, and modified it to reflect correct title